### PR TITLE
Update product detail

### PR DESCRIPTION
**Pull request by ngocnm-0258**
## Related Tickets:
- [#Ticket ID](https://edu-redmine.sun-asterisk.vn/issues/66352)

## WHAT this PR does?
- product detail page

## Notes Impacted Areas:
- N/A.

## Attachments:
image

### CODE REVIEW

**Comment by ngocnm-0258** [comment]
ready

File: resources/views/products/show.blade.php [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+        });
+
+        plusBtn.addEventListener("click", function () {
+            let currentValue = parseInt(quantityInput.value);
+            if (currentValue < {{ $product->number_in_stock }}) {
+                quantityInput.value = currentValue + 1;
+            }
+        });
+    });
+</script>
```
Comment (ghost) [inline resources/views/products/show.blade.php]: ko code js trong blade template. add file js riêng

File: resources/views/products/show.blade.php [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+                </div>
+
+                <div class="mt-12 bg-white p-4 rounded">
+                    <div class="font-bold text-2xl mb-8">{{__('product.show.comment')}}</div>
+                    @foreach ($reviews as $index => $review)
+                        <div class="flex space-x-4 p-4 border-t border-gray-200">
+                            <img class="w-8 h-8" src="https://cdn-icons-png.flaticon.com/512/64/64572.png" alt="user profile">
+                            <div class="flex flex-col">
+                                <div class="text-xl font-bold">{{ $review->user->username }}</div>
+                                <div class="text-base text-gray-500">{{__('product.show.qualityRating')}}{{ $review->rate }}/5</div>
```
Comment (ghost) [inline resources/views/products/show.blade.php]:
```suggestion
                                <div class="text-base text-gray-500">{{ __('product.show.qualityRating') }}{{ $review->rate }}/5</div>
```

File: resources/lang/en/product.php [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+        'sold' => ' Sold',
+        'description' => 'Description',
+        'quantity' => 'Quantity: ',
+        'outOff' => 'Out off ',
+        'productAvailable' => ' products available',
+        'addCart' => 'Add to cart',
+        'buy' => 'Buy now',
+        'qualityRating' => 'Quality rating: ',
+        'message' => 'The restaurant has no other products',
+    ]
```
Comment (ghost) [inline resources/lang/en/product.php]:
```suggestion
    ],
```

**Comment by ngocnm-0258** [comment]
ready

**Review by ghost — APPROVED** [review]

**Pull request merged**